config/samples/infrastructuremanager_v1_gardenercluster.yaml

```diff
@@ -3,21 +3,21 @@ kind: GardenerCluster
 metadata:
   labels:
     kyma-project.io/instance-id: instance-id
-    kyma-project.io/runtime-id: runtime-id
+    kyma-project.io/runtime-id: runtimeid-md-im12
     kyma-project.io/broker-plan-id: plan-id
     kyma-project.io/broker-plan-name: plan-name
     kyma-project.io/global-account-id: global-account-id
     kyma-project.io/subaccount-id: subAccount-id
-    kyma-project.io/shoot-name: shoot-name
+    kyma-project.io/shoot-name: md-im
     kyma-project.io/region: region
-    operator.kyma-project.io/kyma-name: kymaName
-  name: runtime-id
+    operator.kyma-project.io/kyma-name: runtimeid-md-im12
+  name: runtimeid-md-im12
   namespace: kcp-system
 spec:
   shoot:
-    name: shoot-name
+    name: md-im
   kubeconfig:
     secret:
-      name: kubeconfig-runtime-id
+      name: kubeconfig-runtimeid-md-im12
```

Comment:
Are we merging with added `md`?

Reply:
no, adding this was indeed not intentional